contracts/IICO.sol

```diff
@@ -201,7 +202,7 @@ contract IICO {
         uint localSumAcceptedVirtualContrib = sumAcceptedVirtualContrib;
 
         // Search for the cut-off bid while adding the contributions.
-        for (uint it = 0; it < _maxIt && !finalized; ++it) {
+        for (uint it = 0; it < _maxIt ; ++it) {
```

Comment:
Why wouldn't you stop after finalization?

Reply:
Oh thats true, didn't see the state change at the else statement